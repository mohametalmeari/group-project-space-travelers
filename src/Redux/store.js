import { configureStore } from '@reduxjs/toolkit';
import rocketReducer from './Rocket/rocketSlice';
import missionReducer from './features/missions/missionSlice';

const store = configureStore({
  reducer: {
    rocket: rocketReducer,
    mission: missionReducer,
});

export default store;
