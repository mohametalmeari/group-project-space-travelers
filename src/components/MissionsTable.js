import Mission from './Mission';

const MissionsTable = () => (
  <table>
    <thead>
      <tr>
        <th>Mission</th>
        <th>Description</th>
        <th>DescriptionStatus</th>
      </tr>
    </thead>
    <tbody>
      <Mission />
      <Mission />
    </tbody>
  </table>
);
export default MissionsTable;
