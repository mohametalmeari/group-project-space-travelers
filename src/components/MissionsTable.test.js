import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../Redux/store';
import MissionsTable from './MissionsTable';

describe('MissionsTable Component', () => {
  test('Test render of MissionsTable', () => {
    const { container } = render(
      <MemoryRouter>
        <Provider store={store}>
          <MissionsTable />
        </Provider>
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
