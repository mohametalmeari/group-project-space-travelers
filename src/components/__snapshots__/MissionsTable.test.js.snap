// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MissionsTable Component Test render of MissionsTable 1`] = `
<div>
  <div>
    Loading...
  </div>
</div>
`;
